docs/src/@primer/gatsby-theme-doctocat/components/head.js

```diff
@@ -33,7 +36,7 @@ function Head(props) {
       <meta property="og:description" content={description} />
       <meta property="og:image" content={siteMetadata.imageUrl} />
       <meta property="twitter:card" content="summary_large_image" />
-      <script src={primerViewComponentsSrc}></script>
+      <script src={withPrefix(primerViewComponentsSrc)} type="text/javascript"></script>
```

Comment:
```suggestion
      <script deferred src={withPrefix(primerViewComponentsSrc)} type="text/javascript"></script>
```

We need to ensure script runs after DOM is loaded.